lib/gcloud/bigquery/connection.rb

```diff
@@ -462,14 +462,14 @@ def copy_table_config source, target, options = {}
           "configuration" => {
             "copy" => {
               "sourceTable" => {
```

Comment:
the problem is the difference between `projectId` and `project_id`. Now the hash arg could always contain camelcase keys instead, but that isn’t very rubyish, and leaks the API details out of connection into Table